### PR TITLE
Added the capability to run capella reference tests

### PR DESCRIPTION
**Pull request by rolfyone**
- added the `operations/bls_to_execution_change` group to operations that are supported
 - added capella fork support

 partially addresses #5680

Signed-off-by: Paul Harris <paul.harris@consensys.net>

## Documentation

- [X] I thought about documentation and added the `doc-change-required` label to this PR if updates are required.

## Changelog

- [X] I thought about adding a changelog entry, and added one if I deemed necessary.

### CODE REVIEW

File: eth-reference-tests/src/referenceTest/java/tech/pegasys/teku/reference/Eth2ReferenceTestCase.java [hunk 1]
```diff
@@ -70,6 +70,14 @@ public abstract class Eth2ReferenceTestCase {
           .putAll(RewardsTestExecutorBellatrix.REWARDS_TEST_TYPES)
           .build();
 
+  private static final ImmutableMap<String, TestExecutor> CAPELLA_TEST_TYPES =
+      ImmutableMap.<String, TestExecutor>builder()
+          .putAll(TransitionTestExecutor.TRANSITION_TEST_TYPES)
+          .putAll(ForkUpgradeTestExecutor.FORK_UPGRADE_TEST_TYPES)
+          .putAll(RewardsTestExecutorBellatrix.REWARDS_TEST_TYPES)
+          .putAll(OperationsTestExecutor.OPERATIONS_TEST_TYPES)
```
Comment (ajsutton) [inline eth-reference-tests/src/referenceTest/java/tech/pegasys/teku/reference/Eth2ReferenceTestCase.java]: We shouldn't need this as we'll pick it up from the `COMMON_TEST_TYPES`.

File: eth-reference-tests/src/referenceTest/java/tech/pegasys/teku/reference/common/operations/OperationsTestExecutor.java [hunk 2]
```diff
@@ -99,6 +102,10 @@ private enum Operation {
               "operations/execution_payload",
               new OperationsTestExecutor<>(
                   "execution_payload.ssz_snappy", Operation.EXECUTION_PAYLOAD))
+          .put(
+              "operations/bls_to_execution_change",
+              new OperationsTestExecutor<>(
+                  "address_change.ssz_snappy", Operation.EXECUTION_PAYLOAD))
```
Comment (ajsutton) [inline eth-reference-tests/src/referenceTest/java/tech/pegasys/teku/reference/common/operations/OperationsTestExecutor.java]: Wrong operation. :)

File: eth-tests/src/main/java/tech/pegasys/teku/ethtests/TestFork.java [hunk 3]
```diff
@@ -17,4 +17,6 @@ public class TestFork {
   public static final String PHASE0 = "phase0";
   public static final String ALTAIR = "altair";
   public static final String BELLATRIX = "bellatrix";
+
```
Comment (ajsutton) [inline eth-tests/src/main/java/tech/pegasys/teku/ethtests/TestFork.java]: Nit: remove empty line.

**Review by ajsutton — APPROVED** [review]
LGTM. Worth logging a ticket to add block inclusion tests for BLS_TO_EXECUTION_CHANGE once we've implemented the operation pool for them.

File: eth-reference-tests/src/referenceTest/java/tech/pegasys/teku/reference/common/operations/OperationsTestExecutor.java (hunk 2 above)
Comment (rolfyone) [reply eth-reference-tests/src/referenceTest/java/tech/pegasys/teku/reference/common/operations/OperationsTestExecutor.java]: oh that'd make it hard! can't guess what I copied!

**Comment by rolfyone** [comment]
added #6414 when I did the PR, but mentioning here just in reference to your request @ajsutton

**Pull request merged**